packages/perseus/src/renderer.tsx

```diff
@@ -1727,6 +1731,26 @@ class Renderer extends React.Component<Props, State> {
         return this.widgetIds;
     };
 
+    /**
+     * Returns a JSON representation of the content and widgets
+     * that can be passed to an LLM for prompt context.
+     */
+    getPromptJSON(): RendererPromptJSON {
+        const {content, widgets} = this.props;
+        const widgetJSON = {};
+
+        Object.keys(widgets).forEach((key) => {
+            const widget = this.getWidgetInstance(key);
```

Comment:
I'm not sure how much of an impact this would have, but this would be more consistent with `getUserInput`.

```suggestion
    getPromptJSON(): RendererPromptJSON {
        const {content} = this.props;
        const widgetJSON = {};

        this.widgetIds.forEach((id: string) => {
            const widget = this.getWidgetInstance(id);
```